package.json

```
@@ -2,7 +2,8 @@
   "name": "@coremedia/ckeditor5",
   "version": "1.0.0",
   "engines": {
-    "node": ">=14.0.0"
+    "node": ">=14.0.0",
```

```suggestion
    "node": ">=14.19.0",
```

PNPM 7 requires at least node version 14.19. We may want to define it here.